### PR TITLE
ReadPDFFileV2 fix

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-17833

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/CommonScripts/Scripts/ReadPDFFileV2/ReadPDFFileV2.py [hunk 1]
```diff
@@ -494,7 +494,7 @@ def main():
                 # Get metadata:
                 metadata = get_pdf_metadata(cpy_file_path, user_password)
 
-                if user_password:  # The PDF is encrypted
+                if user_password or 'yes' in metadata.get('Encrypted'):  # The PDF is encrypted or has AES encryption
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/Scripts/ReadPDFFileV2/ReadPDFFileV2.py:497]: Does this decrypt the pdf?

Comment (Shellyber) [reply Packs/CommonScripts/Scripts/ReadPDFFileV2/ReadPDFFileV2.py:497]: Yes. The pikepdf package knows how to handle this encryption.

**Review by ShahafBenYakir — COMMENTED** [review]
Nice.

**Comment by Shellyber** [comment]
Link to the passing tests - https://app.circleci.com/pipelines/github/demisto/content/205119/workflows/0aab0524-b898-463d-8520-237ed7eda424/jobs/594508

I'll remove the tests I added and the added file since it's break validation because of this issue: https://jira-hq.paloaltonetworks.local/browse/CIAC-4473 (and it will break the master if merged).

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**